convert-llama-hf-to-gguf.py

```diff
@@ -9,10 +9,12 @@
 import numpy as np
 import torch
 
-from typing import Any, List, Optional
+from typing import Any, List, Optional, TypeAlias
 from pathlib import Path
 from sentencepiece import SentencePieceProcessor
 
+from convert import SpecialVocab
```

Comment:
Can we move SpecialVocab to `gguf.py`?
The way it is, the sample convert scripts depend on the main `convert.py` script which is not great

Reply:
Absolutely. I was thinking about that but I wasn't sure if you'd want it there or not. I'll make the change, possibly not today though.